doc/users/next_whats_new/option_between_for_step_fill_between.rst

```diff
@@ -0,0 +1,17 @@
+step() and fill_between() take a new option where/step="between"
+------------------------------------------------------------------------
+
+Previously one would need to trick step() and fill_between() to plot
+data where x has one point than y, typically when plotting pre-binned
```

Comment:
one *more* point

I think one should sell this more from the point that we now have two new draw-styles. Then in order to use them one can use plot, step, fill_between... 

At the end, this should probably also get a minimal runnable code and picture, so people can envision this change. And also a link to the respective complete examples.

Reply:
I am not sure I get this. The drawstyles exist specifically to enhance the step, fill_between methods. The picture should be taken care of be having the example right?

Reply:
You can refer to the previous what's new section https://matplotlib.org/3.1.1/users/whats_new.html and compare to its source
https://raw.githubusercontent.com/matplotlib/matplotlib/master/doc/users/prev_whats_new/whats_new_3.1.0.rst to see what I mean. It should be much less complicated than the example in the examples section and just show the new drawstyles.

I think the drawstyle exists on its own. You can use `plot`, `step` or `fill_between` to use it. And that's how I would tell the story.